GitHubComments.ps1

```diff
@@ -96,6 +109,11 @@ function Get-GitHubComment
         [ValidateSet('asc', 'desc')]
         [string] $Direction,
 
+        [ValidateSet('raw', 'text', 'html', 'full', '')]
+        [string] $MediaType,
```

Comment:
If `raw` is the default, you can specify that as a default value for this parameter and remove the empty string case:
```powershell
[ValidateSet('raw', 'text', 'html', 'full')]
[string] $MediaType = 'raw'
```